gobblin-core/src/main/java/gobblin/source/extractor/extract/kafka/KafkaRecord.java

```diff
@@ -0,0 +1,87 @@
+package gobblin.source.extractor.extract.kafka;
+
+public class KafkaRecord {
```

Comment:
I am actually using this domain class in another application that reads these generated json files to write them back to Kafka. It would be very convenient if I (or any other application that would read them) didn't have to define it again.

I think it could also be utilised by a future `KafkaDataWriter` as well?